src/common/clipboard.c

```diff
@@ -146,6 +146,20 @@ void guac_common_clipboard_operational_send(guac_client* client) {
 
 }
 
+static void* __send_user_clipboard_remote_update(guac_user* user, void* data) {
+
+    guac_protocol_send_clipboard_remote_update(user->socket);
+
+    return NULL;
+
+}
+
+void guac_common_clipboard_remote_update(guac_client* client) {
+
+    guac_client_log(client, GUAC_LOG_DEBUG, "Sending remote clipboard update to owner");
+    guac_client_for_owner(client, __send_user_clipboard_remote_update, NULL);
```

Comment:
What does `owner` mean here? Is this the clipboard owner?
what does `guac_client_foreach_user` mean then?